account-kit/smart-contracts/src/ma-v2/modules/single-signer-validation/module.ts

```diff
@@ -0,0 +1,34 @@
+import { encodeAbiParameters, type Address, type Hex } from "viem";
+
+export const SingleSignerValidationModule = {
+  encodeOnInstallData: (args: { entityId: number; signer: Address }): Hex => {
+    const { entityId, signer } = args;
+
+    return encodeAbiParameters(
+      [
+        {
+          type: "uint32",
+          value: entityId,
```

Comment:
This `value` field isn't part of the API of `encodeAbiParameters`. Normally you would be passing a `name` and `type` field for each parameter (where I think the name is only used for error messages). See [Viem's docs](https://viem.sh/docs/abi/encodeAbiParameters).

I'm surprised this wasn't a type error, not sure why Viem defined the signature of `encodeAbiParameters` to allow `unknown` params.